src/kernel/bitmap.zig

```diff
@@ -547,24 +566,40 @@ test "Comptime indexToBit" {
 }
 
 test "Comptime setContiguous" {
-    var bmp = ComptimeBitmap(u15).init();
+    var bmp = ComptimeBitmap(u16).init();
```

Comment:
How come this is now a u16?

Reply:
I believe the reason was that it gave me more bits to manipulate and test with.

Reply:
Was just wondering, but cool.